examples/parsl/colmena_rp/colmena_rp.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
```

Comment:
I think this file (and related files under `example/parsl/`) should move to where-ever the executor ends up.